From 2ad692337f05732c6e6432e790127e7c354c91da Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Thu, 12 Dec 2013 11:41:08 +0200
Subject: [PATCH 1/3] Remove P2P support, for now

P2P is removed until driver and firmware of bcm4329 are upgraded to support it.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 hostapd/Android.mk        |    2 +-
 wpa_supplicant/Android.mk |    6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/hostapd/Android.mk b/hostapd/Android.mk
index 2765af8..2bf1df1 100644
--- a/hostapd/Android.mk
+++ b/hostapd/Android.mk
@@ -25,7 +25,7 @@ L_CFLAGS += -DVERSION_STR_POSTFIX=\"-$(PLATFORM_VERSION)\"
 L_CFLAGS += -DANDROID_LOG_NAME=\"hostapd\"
 
 ifeq ($(BOARD_WLAN_DEVICE), bcmdhd)
-L_CFLAGS += -DANDROID_P2P
+#L_CFLAGS += -DANDROID_P2P
 endif
 
 ifeq ($(BOARD_WLAN_DEVICE),$(filter $(BOARD_WLAN_DEVICE), qcwcn UNITE))
diff --git a/wpa_supplicant/Android.mk b/wpa_supplicant/Android.mk
index b8647a6..72dfec5 100644
--- a/wpa_supplicant/Android.mk
+++ b/wpa_supplicant/Android.mk
@@ -30,8 +30,8 @@ L_CFLAGS += -DCONFIG_NO_ROAMING
 endif
 
 ifeq ($(BOARD_WLAN_DEVICE), bcmdhd)
-L_CFLAGS += -DANDROID_P2P
-L_CFLAGS += -DP2P_CONCURRENT_SEARCH_DELAY=0
+#L_CFLAGS += -DANDROID_P2P
+#L_CFLAGS += -DP2P_CONCURRENT_SEARCH_DELAY=0
 endif
 
 ifeq ($(BOARD_WLAN_DEVICE),$(filter $(BOARD_WLAN_DEVICE), qcwcn UNITE))
@@ -71,7 +71,7 @@ INCLUDES += $(LOCAL_PATH)/src/tls
 INCLUDES += $(LOCAL_PATH)/src/utils
 INCLUDES += $(LOCAL_PATH)/src/wps
 INCLUDES += external/openssl/include
-INCLUDES += system/security/keystore
+INCLUDES += system/security/keystore/include
 ifdef CONFIG_DRIVER_NL80211
 INCLUDES += external/libnl-headers
 endif
-- 
1.7.9.5

